jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/RunCommandDialog.kt

```diff
@@ -93,7 +105,7 @@ class RunCommandDialog(private val project: Project, private val container: Cont
     private fun runCommand() {
         try {
             val awsCliPath = AwsCliExecutable().resolve() ?: throw IllegalStateException(message("executableCommon.missing_executable", "AWS CLI"))
-            val execCommand = buildExecCommandConfiguration(command, awsCliPath)
+            val execCommand = buildExecCommandConfiguration(("\"" + command + "\""), awsCliPath)
```

Comment:
Can be cleaned up as `'"' + command + '"'`

Reply:
done